### PR TITLE
Writing style article

### PR DESCRIPTION
**Pull request by magicznyleszek**

### CODE REVIEW

File: docs/_articles/writing-style.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+The rules are:
+
+- Capitalize first letter of everything - from single words to sentences.
+- Finish with a period only if it's a full sentence - e.g. not if it's a heading, short instruction, etc.
+- Always make punctuation part of the translated string (so that RTL languages don't start with a period, for example).
+- Use real ellipsis character (<code>…</code>) rather than three periods (<code>...</code>)<sup>1</sup>.
+- Remember that our name is capitalized like this: `KoboToolbox`.
+
+If in doubt, please mimick what is already there.
```
Comment (jnm) [inline docs/_articles/writing-style.md:13]: No "k" in the standard spelling of "mimic"

File: docs/_articles/writing-style.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+<small><sup>1</sup> In MacOS use `cmd` + `;`. In Windows it's `alt` + `0` `1` `3` `3` on the numberpad. Microsoft Office applications use `ctrl` + `alt` + `.`.</small>
```
Comment (jnm) [inline docs/_articles/writing-style.md:15]: "Are you sure it's `cmd` and not `alt` (AKA `option`)?", says the non mac user. I guess it varies based on keyboard layout, but I remember using `alt` for special characters (and apparently there's a band named after one such shortcut).
http://hints.macworld.com/article.php?story=20081013021724944

Comment (jnm) [reply docs/_articles/writing-style.md:15]: > On Linux, press `<compose key>` + `.` followed by `.`, or, if you do not have a compose key configured, press `ctrl` + `shift` + `u`, release, type `2`, `0`, `2`, `6`, and then press `enter`.

File: docs/_articles/writing-style.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: UI text writing style
+---
+
+The rules are:
+
+- Capitalize first letter of everything - from single words to sentences.
+- Finish with a period only if it's a full sentence - e.g. not if it's a heading, short instruction, etc.
+- Always make punctuation part of the translated string (so that RTL languages don't start with a period, for example).
```
Comment (jnm) [inline docs/_articles/writing-style.md:9]: let's remember to update this documentation when we switch to string identifiers in the UI code

**Review by jnm — CHANGES_REQUESTED** [review]

File: docs/_articles/writing-style.md (hunk 1 above)
Comment (magicznyleszek) [reply docs/_articles/writing-style.md]: Fixed :ok:

File: docs/_articles/writing-style.md (hunk 2 above)
Comment (magicznyleszek) [reply docs/_articles/writing-style.md]: You're right, I use it so often I didn't even check what keys I press :-D I also added the linux info :)

File: docs/_articles/writing-style.md (hunk 3 above)
Comment (magicznyleszek) [reply docs/_articles/writing-style.md:10]: I've added a TODO_NOTE :)

**Pull request merged**